app/models/promo_registration.rb

```diff
@@ -30,6 +30,7 @@ class PromoRegistration < ApplicationRecord
   validates :referral_code, presence: true, uniqueness: { scope: :promo_id }
 
   scope :unattached, -> { where(kind: UNATTACHED) }
+  scope :channel, -> { where(kind: CHANNEL) }
```

Comment:
I'm open to a different name to this scope.  Maybe `channel_owned` might be more intuitive

Reply:
How about `channel_promo`?

Reply:
I think `PromoRegistration.channel_registration` would probably be more correct if we want to go that direction

Reply:
Or `channels_only`? `channel_registration` seems verbose.

Reply:
I like `channels_only`, will change

Reply:
should I change `unattached` to `unattached_only` for consistency?